spec/ameba/rule/lint/unused_class_variable_access_spec.cr

```diff
@@ -0,0 +1,44 @@
+require "../../../spec_helper"
+
+module Ameba::Rule::Lint
+  subject = UnusedClassVariableAccess.new
+
+  describe UnusedClassVariableAccess do
```

Comment:
```suggestion
  describe UnusedClassVariableAccess do
    subject = UnusedClassVariableAccess.new

```

Reply:
I swear there must be another spec file in the codebase with this issue as I keep just copy / pasting other spec files as templates and changing the class name lol.

Reply:
Quite likely, I'm gonna post the PR with fixes which will deal with that once and for all ;)